### PR TITLE
Add Preferences Panel

### PR DESCRIPTION
**Pull request by hharnisc**
### Purpose

Create the new preferences panel, which contains user preferences.

### Notes

https://paper.dropbox.com/doc/Settings-Tab-System-9wjc5pvQerTVDwggWQiIP

### Review

Coming soon :smile:

### CODE REVIEW

**Comment by hharnisc** [comment]
publish-preferences

File: packages/profile-sidebar/index.js [hunk 1]
```diff
@@ -18,7 +18,7 @@ export default connect(
       if (profile.id !== ownProps.profileId) {
         dispatch(push(generateProfilePageRoute({
           profileId: profile.id,
-          tabId: ownProps.tabId,
+          tabId: ownProps.tabId || 'queue',
```
[the remark this answers is not in the document]
Comment (hharnisc) [reply packages/profile-sidebar/index.js]: Great one @msanroman :raised_hands: I'll move this as a default argument :+1:

**Comment by hharnisc** [comment]
Merging this one :100:

File: packages/preferences/.storybook/preview-head.html [hunk 2]
```diff
@@ -0,0 +1,7 @@
+<link href="https://fonts.googleapis.com/css?family=Open+Sans:300,400,600,700,800" rel="stylesheet">
```
Comment (hamstu) [inline packages/preferences/.storybook/preview-head.html]: With the new font from newer `buffer-components`, this should probably be Roboto.

Comment (hamstu) [inline packages/preferences/.storybook/preview-head.html]: With the new font from newer `buffer-components`, this should probably be Roboto.

**Pull request merged**